brave/brave-core
# Fixes contribution that is split

Resolves https://github.com/brave/brave-browser/issues/8828

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Enable rewards
- Claim grant (20 BAT)
- Connect KYC uphold
- Set AC to 50 BAT and add verified publishers to AC table
- Wait for AC to be triggered
- Make sure that 20 BAT was taken from grant and 30 BAT from Uphold

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
